p2p/chain.py

```diff
@@ -720,6 +610,138 @@ class RegularChainSyncer(FastChainSyncer):
         return head.block_number
 
 
+class PeerRequestHandler:
+
+    def __init__(self, db: 'AsyncHeaderDB', logger: TraceLogger, token: CancelToken) -> None:
+        self.db = db
+        self.logger = logger
+        self.cancel_token = token
+
+    async def handle_get_block_bodies(self, peer: ETHPeer, block_hashes: List[Hash32]) -> None:
```

Comment:
Since the class has the name `Handler` what do you think about dropping the `handle_` prefix for these methods?

Reply:
Well, if I drop the `handle_` prefix the names will kinda suggest the methods are getters (e.g. `get_block_bodies()`), so I'd rather keep the prefix